lib/access_check_filter.js

```diff
@@ -64,6 +67,12 @@ module.exports = (hyper, req, next, options, specInfo) => {
                 }
 
                 return contentPromise.then(content => redirect(content, location, options));
+            } else if (content.status === 302) {
+                // Parsoid.js emits a redirect when the new content is parsed because
+                // there could be a race condition with updating a restriction table.
```

Comment:
Is this about the time window between a new restriction being created (ex: revision suppressed), and the update for this being processed in ChangeProp?

Reply:
Yup.